src/App.js

```diff
@@ -104,6 +105,9 @@ export default function App() {
                 <Route path="/bean">
                   <BeanCreate />
                 </Route>
+                <Route path="/shop">
```

Comment:
```suggestion
                <Route path="/shop/:id">
```

URLに表示するshopのshopIdを指定できるようにしてください．
https://reactrouter.com/web/api/Hooks/useparams